packages/slate/src/changes/on-history.js

```diff
@@ -30,6 +30,12 @@ Changes.redo = (change) => {
 
   // Replay the next operations.
   next.forEach((op) => {
+    // When the operation mutates selection, omit its `isFocused` props to
+    // prevent editor focus changing during continuously redoing.
+    const { type, properties } = op
+    if (type === 'set_selection') {
+      delete properties.isFocused
```

Comment:
@doodlewind can you change the logic for these to use `lodash/omit` instead, so we aren't accidentally mutating the `op` object. Thanks!

Reply:
Fixed!